JSDemos/Demos/Charts/Palette/description.md

```diff
@@ -1 +1,21 @@
-The PieChart, like other DevExtreme Data Visualization JavaScript components, provides several built-in palettes. In this demo, you can use the drop-down menus under the PieChart to choose a palette and change how it is extended when it has an insufficient number of colors.
+The PieChart, like other DevExtreme Data Visualization components, allows you to apply multiple predefined palettes. In this demo, you can use two drop-down menus under the PieChart to choose a palette and change the way to extend it when its number of colors is insufficient to paint each series point differently.
+
+## Specify a Palette
+
+A palette is a set of colors that mix well with each other. To apply the needed color scheme, you can assign it to the [palette](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#palette) property. It accepts either the name of a [predefined palette](/Documentation/Guide/Themes_and_Styles/SVG-Based_Components_Customization/#Palettes) or an array of colors. In this demo, you can use the "Palette" selector to apply a new set of colors.
```

Comment:
`that mix well with each other` I'm not sure this is an accurate description of a palette since it is more about the range of colors, perhaps even complementing colors